lib/.gitignore

```diff
@@ -5,3 +5,7 @@
 
 # Do not include tsc source maps
 **/*.js.map
+
+# Do not include .js files from .ts files
+util/version.js
+dialects/index.js
```

Comment:
this causes files not to be published either, I'd rsther have them committed

Reply:
Ohhhh! Good to know. There's a script generating these `.gitignore` items . I can update it stop doing that and check in the generated code.

Reply:
it's ok to generate them prepublish too, to protect against someone forgetting to run it